src/app/authentication/log-in-modal/log-in-modal.component.ts

```diff
@@ -41,6 +41,7 @@ export class LogInModalComponent implements OnInit {
             this.loginService.performSocialLogin(res.provider, res.authToken)
                 .subscribe(backendResponse => {
                     localStorage.setItem('access_token', (<any>backendResponse).access_token);
+                    localStorage.setItem('refresh_token', (<any>backendResponse).refresh_token);
```

Comment:
This will be strongly typed in the future after my code refactoring where we'll have strongly typed responses from our backend API, but it's ok for now :)